sway-lib-std/src/math.sw

```diff
@@ -84,13 +87,14 @@ impl Power for u8 {
     }
 }
 
-// Trait for exponential functions
+/// Trait for exponential functions.
 // Should exist for UFP64, UFP128 and their signed versions
```

Comment:
Not sure if this intended but, should this also be a doc comment?

```suggestion
/// Should exist for UFP64, UFP128 and their signed versions
```

Reply:
I think we should.

```suggestion
/// This should exist for UFP64, UFP128 and their signed versions.
```

Reply:
Hmm, that is a good question. I think the way it is right now might not be entirely suitable for exposing as a doc comment but it doesn't matter that much. I guess we can refactor them later to be more like proper standard library documentation?

If you prefer we can address these stuff later module by module? That could be easier to manage, for example we also have a `Logarithm` trait in the same file which does not have any doc comments on it. If we do a pass over the modules it is easier to catch those stuff. But totally your call, I am good with this either way 🚀